src/components/Header.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Link, Outlet } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { setHeader } from '../redux/headerReducer';
+import load from '../assets/loading.gif';
+import './Header.css';
+
+function Header() {
```

Comment:
- [ ] The header styling does not look different from the previous one. Here is how it looks now. Consider adding some paddings to the nav items

![Screenshot from 2022-04-16 11-18-40](https://user-images.githubusercontent.com/46388113/163673139-625fc13d-c505-4108-b2a7-5da84132d736.png)

### Mobile screen

![Screenshot from 2022-04-16 11-23-02](https://user-images.githubusercontent.com/46388113/163673210-6b000b07-b7ca-4958-8484-3e1a302c50c9.png)